### ISSUE TITLE
MP - two handed weapon is rendered incorectlly for other players 

### ISSUE THREAD

**Issue by m-hynek** [ZioPao/The-Only-Cure#119]
**Describe the bug**
![image](https://github.com/user-attachments/assets/04940992-b594-49eb-b6a5-9c44474e60ac)

**To Reproduce**
- have another player
- cut their right hand and put on prosthetic
- have them equip two handed weapon like baseball bat

**Logs**
Nothing there

**Additional context**
Tried with and without fancy+brutal with same result

**Comment by ZioPao** [comment]
Wait, just to be sure that I'm understanding this correctly before dwelving into debugging, are you using a single baseball bat and it shows as two?

**Comment by m-hynek** [comment]
Yes that is the case. Not only baseball bats, but even brooms and basicaly every two handed item is duplicated, seems it doesnt matter what hand is amputated. One player just reported some weid behavior when holding gas can and refueling, but im confused by their details, will test a bit and get back here with more details

**Comment by m-hynek** [comment]
> Wait, just to be sure that I'm understanding this correctly before dwelving into debugging, are you using a single baseball bat and it shows as two?

you can see yourself correctly, others see you with duplicated weapon

**Comment by ZioPao** [comment]
Ok, I confirmed it on my end. It's related to Brutal\Fancy Handwork though, not TOC, and it happens only when you cut off the left limb, not the right one. I'll see if it's fixable on my end, but no promises

**Comment by m-hynek** [comment]
My bad, on the screen there is left hand prosthetic. But also fancy and brutal are deactivated.

**Comment by ZioPao** [comment]
I didn't manage to reproduce it anymore, it happened only twice, but after unequipping and re-requipping it stopped. I seem to be missing something.

**Comment by m-hynek** [comment]
Maybe when other player equips weapon offscreen. When they already weild weapon when cllient "renders" them, it missing something that hapends clientside in TOC. If it makes sense. Is there some modified Equip item/weapon action? That would make sense because that code would run for the other player in your client only if you see them right? Just blind shot, I dont know yet how everyting works in TOC

**Comment by m-hynek** [comment]
Or there is difference equipping from hotbar and trough contextmenu. Anyway when that amputee player come online, we will do some testing and I will get back here with some more observations

**Comment by m-hynek** [comment]
Hmm I was onto something.  We did some tests. 

me: observing
player2: equiping and unequiping, prosthetic left hand

test 1
weapon: crowbar
weapon in inventory/ground (same result)
player add it to Back slot (hotbar 1)
player equips crowbar trough hotbar 1
result: OK

test 2
weapon: crowbar
weapon in inventory/ground (same result)
player right clicks the weapon on ground/inventory, and in context menu selects "equip in both hands"
result: FAIL - weapon is duplicated, looks like dual wielding (original screenshot)

test 3
weapon: sledgehammer
weapon in inventory/ground (same result)
hotbar method
reult: OK

test 4
weapon: sledgehammer
weapon in inventory/ground (same result)
context menu method
reult: FAIL, even weirder
(player is holding right click to aim)
![image](https://github.com/user-attachments/assets/374ae406-7729-4fbb-8805-7f53fd47bfdd)

test 4
weapon: crowbar
weapon in inventory/ground (same result)
context menu method, but main or off hand instead of both hands
reult: OK

**Comment by cainaazevedoO** [comment]
Hello! did you guys resolved this issue? I just expperienced this but with firearms (We only use Britas), when equipping a Handgun in the secondary slot, and then pressing the Holster shortcut (4 or 5), The character should put the handgun on the left hand away and equip only the holstered weapon but for my friend I'm holding both of them (the "secondary" handgun bugs and shows as equipped on the secondary slot still
Also with two handed firearms I'm holdin two AK's or to Snipers (AWP)
![Double AK](https://github.com/user-attachments/assets/37a7d9bb-3240-4783-8876-6e694d951449)
![Double Sniper](https://github.com/user-attachments/assets/47f223e6-4205-4a14-8fac-5569e10404ec)

**Comment by cainaazevedoO** [comment]
After some testing, we reproduced the issues with equiping throught the context menu in two handed weapons
